sdk/schemaregistry/azure-schemaregistry/README.md

```diff
@@ -1,18 +1,142 @@
 # Azure Schema Registry client library for Python
 
-Azure Schema Registry is a ...
-<<This library is under development>>
+Azure Schema Registry is a service that provides the ability to store and retrieve different types of schemas such as
+Avro, Json, etc.
 
 ## Getting started
 
+### Install the package
+
+Install the Azure Service Bus client library for Python with [pip][pip]:
+
+```Bash
+pip install azure-schemaregistry
+```
+
+### Prerequisites: 
+To use this package, you must have:
+* Azure subscription - [Create a free account][azure_sub]
+* Azure Schema Registry
+* Python 2.7, 3.5 or later - [Install Python][python]
+
+### Authenticate the client
+Interaction with Schema Registry starts with an instance of SchemaRegistryClient class. You need the endpoint and AAD credential to instantiate the client object. 
+
+**Create client using the azure-identity library:**
+
+```python
+from azure.schemaregistry import SchemaRegistryClient
+from azure.identity import DefaultAzureCredential
+
+credential = DefaultAzureCredential()
+endpoint = '<< ENDPOINT OF THE SCHEMA REGISTRY >>'
+schema_registry_client = SchemaRegistryClient(endpoint, credential)
+```
+
 ## Key concepts
 
+- Schema: Schema is the organization or structure for data.
+
 ## Examples
 
+The following sections provide several code snippets covering some of the most common Schema Registry tasks, including:
+
+- [Register a schema](register-a-schema)
+- [Get the schema by id](get-the-schema-by-id)
+- [Get the id of a schema](get-the-id-of-a-schema)
+
+### Register a schema
+
+```python
+import os
+
+from azure.identity import DefaultAzureCredential
+from azure.schemaregistry import SchemaRegistryClient
+
+token_credential = DefaultAzureCredential()
+endpoint = os.environ['SCHEMA_REGISTRY_ENDPOINT']
+schema_group = "<your-group-name>"
+schema_name = "<your-schema-name>"
+serialization_type = "Avro"
+schema_content = """
+{"namespace": "example.avro",
+ "type": "record",
+ "name": "User",
+ "fields": [
+     {"name": "name", "type": "string"},
+     {"name": "favorite_number",  "type": ["int", "null"]},
+     {"name": "favorite_color", "type": ["string", "null"]}
+ ]
+}
+"""
+
+schema_registry_client = SchemaRegistryClient(endpoint=endpoint, credential=token_credential)
+with schema_registry_client, token_credential:
```

Comment:
You actually don't use token_credential inside this with?

Reply:
nice catch, I forget to remove it here